src/main/java/net/earthcomputer/clientcommands/command/MinesweeperCommand.java

```diff
@@ -290,6 +297,14 @@ private boolean isWithinBounds(int x, int y) {
             return 0 <= x && x < boardWidth && 0 <= y && y < boardHeight;
         }
 
+        private Stream<Vector2i> getNeighbours(int x, int y) {
+            return Stream.of(
+                new Vector2i(x - 1, y - 1), new Vector2i(x, y - 1), new Vector2i(x + 1, y - 1),
+                new Vector2i(x - 1, y),                                    new Vector2i(x + 1, y),
```

Comment:
```suggestion
                new Vector2i(x - 1, y), new Vector2i(x + 1, y),
```
What is this ungodly amount of whitespace for?

Reply:
The neighbour tiles are the 8 surrounding, I divided the lines such that the position of the `Vector2i` is also the position relative to the `x` and `y` coordinates of the tile to get the neighbours of, although now I see that IntelliJ screwed me up a lil bit with the `x:` and `y:` so I'll edit it to be corrected for when not viewing with those parameter hints.